### DISCUSSION TITLE
scrum prep Milestone 3 - 3

### DISCUSSION THREAD

**Discussion by gabrielpadilla7** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#203] [General]
Write your scrums here

**Comment by gabrielpadilla7** [comment]
Maintained documentation, verified documentation work and structure, closed issues

**Comment by MrL9000** [comment]
Fixed the font on the Vehicle menu, in this weekend I'll set up a specification task and start work on both of my other tasks.

**Comment by AlexisTraversoPR** [comment]
Implemented a footer containing links to the most relevant websites of the team behind the project

**Comment by Brandell13** [comment]
This week I was working with two documentation tasks

**Comment by JaseyAcevedo** [comment]
Working on ltt and exams and reports.

**Comment by Janiel777** [comment]
Esta semana hice un LTT de refactorizacion para separar lógicas de UI y business para un mejor mantenimiento en un futuro. #201

**Comment by juan-ortiz92** [comment]
This sprint I worked on my third lecture topic task and on documentation. I also studied this week to work on my final lecture topic task.

**Comment by Yahid1** [comment]
Closed some Issues, and ensured the validity of tasks as well as help to understand some tasks.

**Comment by IvanierB** [comment]
This sprint I worked with two lecture topic tasks. I am also waiting for an issue to be resolved regarding a ltt that I did in milestone 2 but hasn't been accounted for.

**Comment by rafa-quiles** [comment]
Working on documentation and lecture topic task

**Comment by AleDiazz** [comment]
Worked on ltt and helped improve documentation based on professors feedback

**Comment by sebasagir** [comment]
worked on a documentation task

**Comment by dAninAd1248** [comment]
Replaced google maps API key

**Comment by marcosarraiza** [comment]
Worked on changing the colors of the vehicle tab and comminucated with a team member about LTT's.

**Comment by Alej24** [comment]
Finished a Lecture Topic Task as well as another task.  Am researching to do another LTT and to finish another task I have assigned.

**Comment by JConcepcion01** [comment]
I worked on my tasks and getting the documentation ready for others to start working on it for milestone 3

**Comment by diegorobles2** [comment]
This week I was working with my documentation task.

**Comment by yamayrie** [comment]
i worked on a lecture topic task. i've also been working on fixing documentation based on feedback and been thinking of another ltt topic

**Comment by JoelTorres20** [comment]
This week I drafted my next LTT and will create it as a task later this weekend.

**Comment by dylanDeJesus21** [comment]
Fixed Domain, Interface, and Machine requirements
